### DISCUSSION TITLE
Mandrill

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4300] [Q&A] [answered]
Hello, I made the configuration for inbound email with Mandrill but i have this error:

Webhook failed with error: POST to https://help.XXXXX.it/rails/action_mailbox/mandrill/inbound_emails failed with 401:

I don't know how to handle it, i made the configuration following the istructions.

Some could suggest me how to corretct this errore?
The machine is Ubuntu on Google Cloud Platform.

M.

**Comment by vishnu-narayanan** [comment, marked as answer]
Have you configured all the environment variables[1] and enabled transactional emails[2]?

1. https://www.chatwoot.com/docs/self-hosted/configuration/environment-variables#mandrill
2. https://mailchimp.com/help/add-or-remove-transactional-email/

**Reply by mattiamarras** [reply]
Thank you, I think I was using the wrong api key.

**Reply by vishnu-narayanan** [reply]
Cool :)
